internal/storage/bucket_handle_test.go

```diff
@@ -366,6 +367,56 @@ func (t *BucketHandleTest) TestCreateObjectMethodWithGenerationAsZeroWhenObjectA
 	AssertTrue(errors.As(err, &precondition))
 }
 
+func (t *BucketHandleTest) TestCreateChunkUploaderMethodWithValidObject() {
+	chunkSize := googleapi.MinUploadChunkSize
+	uploader, err := t.bucketHandle.CreateChunkUploader(context.Background(),
+		&gcs.CreateObjectRequest{
+			Name: "test_object",
+		},
+		chunkSize,
+		nil,
+	)
+
+	AssertEq(nil, err)
+	AssertNe(nil, uploader)
+	AssertEq(0, uploader.BytesUploadedSoFar())
+}
+
+func (t *BucketHandleTest) TestCreateChunkUploaderMethodWithGenerationAsZero() {
+	var generation int64 = 0
+	chunkSize := googleapi.MinUploadChunkSize
+	uploader, err := t.bucketHandle.CreateChunkUploader(context.Background(),
```

Comment:
can we follow arrange, act and assert here and everywhere go/unit-testing-practices?polyglot=go#structure?

Reply:
I only followed the pattern of the other tests in this file. 
I have updated the 3 chunkUploader tests that I've added. I have followed the arrange,act,assert pattern everywhere in any new files.